google/codelab/provision_spinnaker_codelab_image.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script provisions a fresh spinnaker image with a Jenkins
+# server and an aptly repository, served on nginx. It also
+# provides a basic Jenkins configuration. This is to be used
+# specifically for creating a new (running) codelab image out of a fresh
+# spinnaker image.
+
+if [[ `/usr/bin/id -u` -ne 0 ]]; then
+  echo "$0 must be executed with root permissions; exiting"
+  exit 1
+fi
+
```

Comment:
Do you want to check /opt/spinnaker/pylib to test for an installation?
You depend on it later on.